eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Utilities/ResourceHelpers.cs

```diff
@@ -35,5 +35,18 @@ public static string GetRestClientFieldName(string restClientName)
                 _ => null
             };
         }
+
+        /// <summary>
+        /// Determines if the given resource operation kind should be treated as a "fake LRO" (Long-Running Operation).
+        /// A "fake LRO" is an operation that is not a true long-running operation at the REST API level,
+        /// but is modeled as an LRO in the SDK for consistency and to provide a uniform developer experience.
+        /// Only Create and Delete operations are treated as fake LROs because, in Azure, these operations are
+        /// commonly expected to be long-running, even if the underlying service does not implement the LRO pattern.
+        /// </summary>
+        public static bool IsFakeLroMethod(ResourceOperationKind operationKind)
```

Comment:
This is not really `IsFakeLroMethod` which should check if it is not a LRO, might just be `IsForceLro`